tests/Unit/Commands/RunBackupCommandTest.php

```
@@ -121,4 +122,19 @@ public function test_runs_enabled_backup_even_after_a_failed_run(): void
+    {
+        SSH::fake();
+        Bus::fake();
+        Carbon::setTestNow('2026-06-19 10:00:00');
+
+        $backup = $this->createBackup(['interval' => '0 * * * *', 'server_id' => 999999]);
+
+        $this->artisan('backups:run')
+            ->expectsOutput('0 backups started');
+
+        $this->assertDatabaseHas('backups', ['id' => $backup->id]);
+        Bus::assertNothingDispatched();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Bus' in method 'test_does_not_run_backups_whose_server_is_missing'. (undefined)

(StaticAccess)